decision-process/README.md

```diff
@@ -0,0 +1,77 @@
+---
+type: decision
+stakeholders:
+    - sig-architecture
+    - team-horizon
+    - team-team
+issue:
+---
+
+# RFC Decision Making Process
+
+This RFC follows on from the RFC repository README (see [here](https://github.com/giantswarm/rfc/blob/b3f4d2af6ff8965e542baae7d0e8fadb9163f356/README.md)).
+
+This RFC aims to expand on the current RFC process by adding a lightweight decision making process.
+The aim of this decision making process is to help enable, improve, and streamline technical decision making at Giant Swarm.
+
+## RFC Types
+
+To allow for a decision making process without removing the current free-form nature of RFCs, RFC types are proposed. The following changes will be introduced:
+
+- All RFCs will have a YAML header.
+- The YAML header will contain the following mandatory fields:
+    - `type`
+
+The 'default' type is `general`, and has no other requirements, to allow for free form writing. All currently merged RFCs would fall under this type, and will be moved to this type.
+
+The following is an entire and valid example of a YAML header for an RFC:
+
+```
+---
+type: general
+---
+```
+
+## Decision Making Process
+
+The two main aims of the decision making process are to ensure that the relevant groups have seen and approve the decision, and that the implementation of the decision is being tracked.
```

Comment:
I would like to see some more in depth proposal here - there are multiple issues which keep re-appearing with RFCs:
- How long should an RFC be open until a decision is reached?
- How do you deal with people who do not respond in a reasonable timeframe (what is considered reasonable?)?
- What to do (roughly) if a decision can not be reached? (probably schedule a meeting)

I am fine if you don't want to include this here directly but to me, answers to these questions will be necessary to have a smoother RFC process going forward.

Reply:
yeah, i tried to get some language down for these, but i'm not happy enough with it to make it "official"

generally:
- i don't want to give a specific timeframe / deadline for how long an RFC should be open - i could believe it's very dependent on complexity of the topic / number of participants etc. 
- i think if people do not respond / if decision can't be reached, we need to schedule a meeting, sure. i could believe there's some case-by-case work here until we figure / smooth it out

my general thinking is to use sig arch sync to review / push hanging RFCs - i.e: if we have open decision RFCs, let's bring them up in sig arch and see what we need to do to unblock, and then formalise that going further

sounds good? i know this is non-concrete, but i think we need to live it a bit

Reply:
But IMO we need to start from some point and then refine it. Like @AndiDog I missing here a bit of how should I proceed if I want to take a decision. Andreas also offer to collaborate maybe can give you a hand